foxbot_core.ino

```diff
@@ -221,6 +228,26 @@ void loop() {
 		odom.twist.twist.angular.z = angular_velocity_est;
 
     	odom_publisher.publish(&odom);
+		
+		// Calculate joint_state message
+		motor_left_rate_est= motor_left_rate_est*(-1);
+		left_wheel_position = left_wheel_position + (motor_left_rate_est*(-1)) * WHEEL_RADIUS;
+		right_wheel_position = right_wheel_position + motor_right_rate_est * WHEEL_RADIUS;
+		pos[0] = right_wheel_position;
+		vel[0] = motor_right_rate_est;
+		pos[1] = left_wheel_position;
+		vel[1] = motor_left_rate_est;
+
+		//Feed joint_state message
+		robot_state.header.stamp = nh.now();
+		robot_state.header.frame_id = "";
+		robot_state.name = joint_name;
+		robot_state.position = pos;
+		//robot_state.position = ["debug_left.x", "debug_right.x"];
+		robot_state.velocity = vel;
+		robot_state.effort = eff;
```

Comment:
could be commented for now

Reply:
only the `robot_state.effor = eff;` line
(github won't allow only single line selecting)